app/views/api_docs/pages/release_notes.md

```diff
@@ -1,3 +1,8 @@
+## 9th March
+
+- Update return values of `uk_residency_status` to correspond with candidate application options. Removes 'Candidate does not know' and the value 'Candidate needs to apply for permission to work and study in the UK' is returned instead.
```

Comment:
could we separate the changes in this section as `Changes to existing attributes:` and `New attributes:` as per the previous release notes, although that said there doesn't seem to be a convention ;
🤔